.github/workflows/GuideConverterAction.yaml

```
@@ -0,0 +1,100 @@
+name: GuideConverter
+on: 
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch pushed to'     
+        required: true
+        default: 'defaulted'
+      guide_name:
+        description: 'Guide changed'
+        required: true
+        default: 'not given'
```

```suggestion
        default: 'guide-rest-intro'
```

Add more useful default.

---

Done